app/response_svc.py

```diff
@@ -1,16 +1,115 @@
 from aiohttp_jinja2 import template
+import asyncio
+import json
+import uuid
 
```

Comment:
Imports should go std lib, app imports, 3rd party
https://www.python.org/dev/peps/pep-0008/#imports